core/optaplanner-core-impl/src/main/java/org/optaplanner/core/impl/domain/variable/listener/support/Notification.java

```diff
@@ -61,12 +61,8 @@ static <Solution_> ListVariableNotification<Solution_> elementRemoved(Object ent
         return new ElementRemovedNotification<>(entity, index);
     }
 
-    /**
-     * An element was moved from {@code sourceEntity[sourceIndex]} to {@code destinationEntity[destinationIndex]}.
-     */
-    static <Solution_> ListVariableNotification<Solution_> elementMoved(
-            Object sourceEntity, int sourceIndex, Object destinationEntity, int destinationIndex) {
-        return new ElementMovedNotification<>(sourceEntity, sourceIndex, destinationEntity, destinationIndex);
+    static <Solution_> ListVariableNotification<Solution_> subListChanged(Object entity, int fromIndex, int toIndex) {
+        return new SubListChangedNotification<>(entity, fromIndex, toIndex);
```

Comment:
align naming to the method in ListVariableListener, so probably listVariableChanged(entity, fromIndex, toIndex)